packages/components/tsconfig.json

```diff
@@ -11,12 +11,13 @@
     "moduleResolution": "node",
     "noUnusedLocals": true,
     "outDir": "dist/",
+    "resolveJsonModule": true,
```

Comment:
SvgIcon imports `@chanzuckerberg/eds-tokens/json/variables.json`, and TIL that we need to tell TypeScript to handle that.